internal/proxy/task_insert.go

```diff
@@ -125,6 +125,15 @@ func (it *insertTask) PreExecute(ctx context.Context) error {
 		return merr.WrapErrAsInputError(merr.WrapErrParameterTooLarge("insert request size exceeds maxInsertSize"))
 	}
 
+	info, err := globalMetaCache.GetCollectionInfo(it.ctx, it.insertMsg.GetDbName(), collectionName, 0)
```

Comment:
GetCollectionInfo && GetCollectionSchema twice might not be slow. and why do we need a Baisc Collection info?

Reply:
we can do getFullCollectionInfo so base info and schema is there